src/pages/Signup/Signup.scss

```diff
@@ -1 +1,21 @@
-.Signup {}
```

Comment:
Keep this class and write this pages css insite that class.. in scss we can write nested css 
```scss
.Signup {
     .form {
          width: 100%
     }
}
```
like this..